content/zh/docs/reference/glossary/cloud-controller-manager.md

```diff
@@ -33,18 +33,16 @@ that embeds cloud-specific control logic. The cloud controller manager lets you
 cluster into your cloud provider's API, and separates out the components that interact
 with that cloud platform from components that only interact with your cluster.
 -->
-云控制器管理器是指嵌入特定云的控制逻辑的
+`cloud-controller-manager` 是指嵌入特定云的控制逻辑之
 {{< glossary_tooltip text="控制平面" term_id="control-plane" >}}组件。
-云控制器管理器使得你可以将你的集群连接到云提供商的 API 之上，
+`cloud-controller-manager` 允许你将你的集群连接到云提供商的 API 之上，
 并将与该云平台交互的组件同与你的集群交互的组件分离开来。
 
-<!--more--> 
```

Comment:
This tag is necessary for glossary.

The [preview](https://deploy-preview-34113--kubernetes-io-main-staging.netlify.app/zh/docs/reference/glossary/?fundamental=true&operation=true#term-cloud-controller-manager) is broken.